scrc/dataset_creation/overview_creator.py

```diff
@@ -0,0 +1,64 @@
+from root import ROOT_DIR
+import os
+import sys
+import csv
+from tqdm import tqdm
+from scrc.utils.court_names import get_all_courts
+from scrc.enums.split import Split
+from scrc.utils.main_utils import get_config
+
+csv.field_size_limit(sys.maxsize)
+config = get_config()
+data_dir = f"{str(ROOT_DIR)}/{config['dir']['data_dir']}"
+datasets_subdir = f"{data_dir}/{config['dir']['datasets_subdir']}"
+
+
+#   creates an overview of the size of the generated datasets and exports them in a csv file
+def create_overview(path, export_path, export_name="overview", include_all=False):
```

Comment:
Suggestion: Move this entire function to the dataset_creator. Like this, you already have access to the config and all the directories. It makes everything a bit simpler. We could still refactor out some easy functions into a utils module later

Reply:
Alright